protocol/ntpshm/ntpshm.go

```diff
@@ -62,14 +60,16 @@ func Create() (uintptr, error) {
 	return shmID, nil
 }
 
-func ptrToNTPSHM(shmptr uintptr) *NTPSHM {
-	var shm []NTPSHM
-	sh := (*reflect.SliceHeader)(unsafe.Pointer(&shm))
-	sh.Data = shmptr
-	sh.Len = 1
-	sh.Cap = 1
+func ptrToBytes(shmptr uintptr) []byte {
+	// Slice memory layout
+	var sl = struct {
+		addr uintptr
+		len  int
+		cap  int
+	}{shmptr, 92, 92}
```

Comment:
where do these numbers come from?

Reply:
Fair question. I updated to actually use `NTPSHMSize` which is a standard size of the SHM segment.